lively.morphic/text/morph.js

```diff
@@ -3933,11 +3935,40 @@ export class Text extends Morph {
     }
   }
 
+  showIconButton () {
+    System.import('lively.morphic').then(
+      async (morphic) => {
+        const { PropertyLabel, PropertyLabelHovered } = await System.import('lively.ide/studio/shared.cp.js');
+        const iconButton = morphic.part(PropertyLabel, {
+          tooltip: 'Insert Icon',
+          fontSize: 14,
+          textAndAttributes: Icon.textAttribute('heart-music-camera-bolt')
+        });
+        const iconButtonHolder = new Morph({
+          fill: Color.rgb(30, 30, 30).withA(0.95),
+          borderRadius: 3,
+          layout: new TilingLayout({
+            hugContentsVertically: true,
+            hugContentsHorizontally: true
+          }),
+          submorphs: [
+            iconButton]
+        });
+        iconButton.master = { auto: PropertyLabel, hover: PropertyLabelHovered };
+        this.iconButton = iconButton;
+        this.iconButton.onMouseDown = () => this.execCommand('add icon at cursor position');
+        iconButtonHolder.openInWorld(this.topRight);
+      });
+  }
+
```

Comment:
How about we put this logic into a command that we move into the editor plugin? That way we can get rid of this weird async import stuff.